app/case/server.py

```diff
@@ -64,3 +64,16 @@ def update_status(deed_id):
             return jsonify(case_status=case_status), status.HTTP_200_OK
         else:
             abort(status.HTTP_400_BAD_REQUEST)
+
+    @blueprint.route('/case/<case_id>/deed', methods=['POST'])
+    def update_case_deed(case_id):
+        case = Case.get(case_id)
+
+        if case is None:
+            abort(status.HTTP_404_NOT_FOUND)
+
+        deed_id = request.data['deed_id']
+        case.deed_id = deed_id
+
+        case.save()
+        return str(status.HTTP_200_OK)
```

Comment:
Aye, I was just thinking it would be useful to have confirmation of what you did when doing something. e.g. If I click a button and get "Cool, thanks" back I'll be confused. If I get "You just deleted your car" I'll have more context. Don't mind that much.


**Question though**: Should we be updating the status of the case to show that a deed has been created? I think probably.